samples/index.html

```diff
@@ -14,7 +14,8 @@
         <p class="mt-5 mb-3">
           Select a sample:
         </p>
-        <p><a href="./custom-ui" class="btn btn-primary">Custom UI</a></p>
+        <p><a href="/embedded-login/" class="btn btn-primary">Embedded Login</a></p>
```

Comment:
Rather than "custom UI", I've changed this to "embedded login" to compliment "central login".